Kentor.AuthServices.StubIdp/App_Data/default.json

```diff
@@ -1,5 +1,7 @@
 {
   "HideDetails": false,
+  "EnforceXmlns": false,
```

Comment:
This enforces XS (Xml Schema), not NS (NameSpace). And it is for attributevalues only, which should be more clear. What about `EnforceAttributeXsString`?